api/conf/settings/sentry.py

```diff
@@ -12,10 +12,16 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DJANGO_DEBUG_ENABLED", default=False, cast=bool)
+
+INTEGRATIONS = [
+    DjangoIntegration(),
+    LoggingIntegration(event_level=None, level=None),
```

Comment:
This is what prevents the error being sent to Sentry twice: once with the correct information, and the other - with the logged json as the error name (as in https://openverse.sentry.io/share/issue/e2bc26efc51048a981a0da629a34930d/)

Reply:
Maybe you could just link to https://github.com/kiwicom/structlog-sentry?tab=readme-ov-file#logging-as-json in a comment for documentation? The reason it prevents the duplicate logs is specific and important for context and maintainability of the logging configuration (i.e., that we specifically use JSON logging).